site/_layouts/default.html

```diff
@@ -64,7 +64,7 @@
                 </div>
                 <div class="col-md-9 copyright">
                   <p>
-                    Copyright © 2011-2017 Developer Express Inc.</br>
+                    Copyright © 2011-2018 Developer Express Inc.</br>
```

Comment:
Can we change year automatically, like `{{ site.time | date: '%Y' }}`?

Reply:
Yes, we should